### DISCUSSION TITLE
React help

### DISCUSSION THREAD

**Discussion by netprince1210venus** [community/community#68189] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have some issue with javascript maximum call function

Who met this issue before
Please help me.

**Comment by tp1845** [comment]
Hey
Just pls review your code to find recallback function
And need to check your code.
Thanks

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
